src/init/init.c

```diff
@@ -43,6 +41,8 @@ int master_core_init(struct sof *sof)
 	platform_init_memmap();
 	init_heap(sof);
 
+	interrupt_init();
```

Comment:
I think this one would be better as separate commit. It's not spinlock fix

Reply:
Done.

Reply:
thanks